### PR TITLE
don't add label repeatly

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: web/hook.go [hunk 1]
```diff
@@ -110,10 +110,24 @@ func Hook(c *gin.Context) {
 		log.Errorf("Error remove old labels for %s pr %d. %s", repo.Slug, hook.Issue.Number, err)
 	}
 
-	// add new label
-	err = remote.AddIssueLabels(c, user, repo, hook.Issue.Number, []string{labels[idx]})
+	oriLabels, err := remote.GetIssueLabels(c, user, repo, hook.Issue.Number)
 	if err != nil {
-		log.Errorf("Error add new label for %s pr %d. %s", repo.Slug, hook.Issue.Number, err)
+		log.Errorf("Error remove old labels for %s pr %d. %s", repo.Slug, hook.Issue.Number, err)
```
Comment (tboerger) [inline web/hook.go]: The message is wrong, it failed to get the labels

Comment (lunny) [reply web/hook.go]: Done.

**Comment by tboerger** [comment]
LGTM

**Comment by tboerger** [comment]
We should also add directly after opening a pr the need 2 label

**Comment by lunny** [comment]
Yes, right! I will fix failed removing label first after this PR merged.

**Comment by strk** [comment]
LGTM

**Comment by lunny** [comment]
And we have to check if the PR is closed. For a closed PR, don't add or remove any labels.

**Pull request merged**